src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const largeImage = document.querySelector('#largeImg');
+const smallImages = document.querySelector('#thumbs');
+
+smallImages.addEventListener('click', function(events) {
+  const target = events.target;
+
+  if (target.classList.contains('gallery__thumb')
+  || target.classList.contains('list-item')) {
+    const newSrc = target.parentNode.getAttribute('href');
+
+    largeImage.setAttribute('src', newSrc);
+    events.preventDefault();
+  }
+});
```

Comment:
it's better to prevent default behaviour at the beginning
```suggestion
smallImages.addEventListener('click', function(events) {
  events.preventDefault();
  const target = events.target;

  if (target.classList.contains('gallery__thumb')
  || target.classList.contains('list-item')) {
    const newSrc = target.parentNode.getAttribute('href');

    largeImage.setAttribute('src', newSrc);
  }
});
```